static/src/javascripts/projects/common/utils/drop-while.js

```diff
@@ -0,0 +1,17 @@
+define(function () {
```

Comment:
It would be useful if we could document these utils somewhere (even if just with JSDoc)